screens/authentication/auth.css

```diff
@@ -0,0 +1,77 @@
+@import url(/screens/homepage/homepage.css);
+@import url(/screens/cart/cart.css);
+
+.account-page {
+  padding: 50px 0;
+  background: radial-gradient(var(--offwhite-color), var(--secondary-color));
+}
+.form-container {
+  background: var(--offwhite-color);
+  width: 300px;
+  height: 400px;
+  position: relative;
+  text-align: center;
+  padding: 20px 0;
+  margin: auto;
+  box-shadow: 0 0 20px var(--offwhite-color);
+  overflow: hidden;
+}
+.form-container span {
+  font-weight: bold;
+  padding: 0 10px;
+  color: var(--primary-color);
+  cursor: pointer;
+  width: 100px;
+  display: inline-block;
+}
+.form-btn {
+  display: inline-block;
+}
+.form-container form {
+  max-width: 300px;
+  padding: 0 20px;
```

Comment:
Suggestion: Can use relative values for padding and width.

Reply:
ok